src/__swaps__/screens/Swap/components/CoinRow.tsx

```diff
@@ -111,7 +104,7 @@ export const CoinRow = ({
               <CoinRowButton icon="􀅳" outline size="icon 14px" />
               <CoinRowButton
                 color={isFavorite(address) ? '#FFCB0F' : undefined}
-                onPress={() => toggleFavorite(address)}
+                onPress={() => toggleFavorite(mainnetAddress, chainId)}
```

Comment:
general question, does mainnetAddress exist on tokens that does not exist in mainnet?

Reply:
yes it defaults to the first chain